tests/unittests/OperatorTest.cpp

```diff
@@ -135,6 +135,76 @@ TEST_P(InterpAndCPU, log) {
   }
 }
 
+TEST_P(Operator, logit) {
+  auto eps = 1E-6f;                // the default in Caffe2
+  constexpr std::size_t size = 10; // sample size for randomized tests
+
+  // known values: the logarithm of the odds for 25%, 50%, and 75% probabilities
+  auto *X = mod_.createPlaceholder(ElemKind::FloatTy, {1, 1, 3}, "X", false);
+  ctx_.allocate(X)->getHandle() = {0.25f, 0.5f, 0.75f};
+  auto *LogitX = F_->createLogit("logitX", X, eps);
+  auto *saveX = F_->createSave("saveX", LogitX);
+  auto *savePlaceholderX = saveX->getPlaceholder();
+  ctx_.allocate(savePlaceholderX);
+
+  // differential test:
+  // ensure we match an oracle `logit_test` (a C++ reimplementation test)
+  auto clamp_test = [](float v, float lo, float hi) {
+    return std::max(std::min(v, hi), lo);
+  };
+  auto logit_test = [clamp_test](float x, float eps = 1E-6f) {
+    float p = clamp_test(x, eps, 1.0f - eps);
+    return std::log(p / (1.0f - p));
+  };
+
+  auto *input =
+      mod_.createPlaceholder(ElemKind::FloatTy, {size}, "input", false);
+  // ensure the input data is clamped in (eps, 1 - eps) (as in Caffe2)
+  ctx_.allocate(input)->getHandle().randomize(eps, 1.0f - eps, mod_.getPRNG());
+
+  auto *logitDiff = F_->createLogit("logitDiff", input, eps);
+  auto *saveDiff = F_->createSave("saveDiff", logitDiff);
+  ctx_.allocate(saveDiff->getPlaceholder());
+
+  // property: zero-sum for the log-odds for complementary events probabilities
+  // i.e., logit(p) + logit(1 - p) == 0
+  Node *complInput = input;
+  Node *const1 = F_->createSplat("const1", input->getType(), 1.0);
+  complInput = F_->createSub("sub", const1, complInput);
+  Node *logitCompl = F_->createLogit("logitCompl", complInput, eps);
+  auto *saveCompl = F_->createSave("saveCompl", logitCompl);
+  ctx_.allocate(saveCompl->getPlaceholder());
+
+  // property: the logit function is the right-inverse of the logistic function
+  // i.e., logistic(logit(p)) == p
+  auto logistic_test = [](float x) { return 1.0f / (1.0f + std::exp(-x)); };
+
+  EE_.compile(CompilationMode::Infer, F_, ctx_);
+  EE_.run();
+
+  // results: known values
```

Comment:
Consider removing this set of tests altogether. These tests are basically same as the tests against the oracle.

Reply:
Done!